### PR TITLE
prepares 3.4.0 

### PR DESCRIPTION
**Pull request by bodiyang**

### CODE REVIEW

File: docs/about/releases.md [hunk 1]
```diff
@@ -3,6 +3,25 @@ Release history
...
+(last merged pull request is
+[#2677](https://github.com/PSLmodels/Tax-Calculator/pull/2677))
+
+**This is an enhancement and bug-fix release.**
+
+**API Changes**
+
+**New Features**
+- Tax-Calculator baseline update for CBO economic projections, published in May, "The Budget and Economic Outlook: 2023 to 2033" [[#2676](https://github.com/PSLmodels/Tax-Calculator/pull/2676) by Bodi Yang], with primary work in [[TaxData PR #421](https://github.com/PSLmodels/taxdata/pull/421)by Bodi Yang]
+- Fix calculaton of 2021 child and other dependent credit [[#2677](https://github.com/PSLmodels/Tax-Calculator/pull/2677) by Matt Jensen]
```
Comment (jdebacker) [inline docs/about/releases.md:17]: @bodiyang Can you move these fixes to the bug fixes section below?

File: docs/about/releases.md [hunk 2]
```diff
@@ -3,6 +3,25 @@ Release history
 Go [here](https://github.com/PSLmodels/Tax-Calculator/pulls?q=is%3Apr+is%3Aclosed)
 for a complete commit history.
 
+2023-05-16 Release 3.4.0
```
Comment (jdebacker) [inline docs/about/releases.md:6]: Let's change the date here and below to May 5 (we'll get this merged today).

**Comment by bodiyang** [comment]
@jdebacker thanks Jason!

File: docs/index.md [hunk 3]
```diff
@@ -51,7 +51,7 @@ The cross-model validation work with NBER's TAXSIM-27 model is described
 
 ## Latest release
 
-{doc}`3.3.0 (2022-12-16) <about/releases>`
+{doc}`3.4.0 (2023-05-16) <about/releases>`
```
Comment (jdebacker) [inline docs/index.md:54]: Date change here.

**Comment by jdebacker** [comment]
@bodiyang One more date change.  And then please resolve conflicts from `test_wtqts_benchmark.csv` and i will merge. Thanks!

**Comment by feenberg** [comment]
I wish you would use the latest version - taxsim35. It will accept 
taxsim27 input files, so the change should not be troublesome.

I can send a linux, osx or windows binary.

Daniel Feenberg
617-682-6204

**Comment by jdebacker** [comment]
@feenberg comments:
> I wish you would use the latest version - taxsim35. It will accept
taxsim27 input files, so the change should not be troublesome.

With the recent merge of PR #2619, we do have validation files to work with taxsim35.  For now, we've left in the taxsim27 validation files as well, but will likely remove those in a future release.

**Comment by jdebacker** [comment]
@bodiyang Thanks for this PR.  Merging.

**Pull request merged**